tests/e2e_test/README.md

```diff
@@ -0,0 +1,19 @@
+# E2e Tests
+
+This Subdirectory contains e2e tests used for E2E Verification.
```

Comment:
```suggestion
This subdirectory contains e2e tests used for the e2e verification.
```
the e2e verification of what? Could we add what is verified?

Reply:
The watcher component is what is being verified. I will add it 👍